.travis.yml

```diff
@@ -0,0 +1,22 @@
+ # Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Sure, will discuss in the @dev channel. I can remove it from PR and can add later as part of different PR.